nodejs-babel-core/nodejs-babel-core.spec

```diff
@@ -1,194 +1,162 @@
-Provides: bundled-npm(globals) = 8.18.0
-Provides: bundled-npm(lodash) = 4.14.0
-Provides: bundled-npm(chalk) = 1.1.3
+Provides: bundled-npm(babylon) = 6.18.0
+Provides: bundled-npm(private) = 0.1.8
+Provides: bundled-npm(source-map) = 0.5.7
 Provides: bundled-npm(esutils) = 2.0.2
-Provides: bundled-npm(js-tokens) = 2.0.0
-Provides: bundled-npm(core-js) = 2.4.1
-Provides: bundled-npm(home-or-tmp) = 1.0.0
+Provides: bundled-npm(js-tokens) = 3.0.2
+Provides: bundled-npm(chalk) = 1.1.3
+Provides: bundled-npm(lodash) = 4.17.4
+Provides: bundled-npm(home-or-tmp) = 2.0.0
 Provides: bundled-npm(mkdirp) = 0.5.1
-Provides: bundled-npm(core-js) = 1.2.7
-Provides: bundled-npm(detect-indent) = 3.0.1
-Provides: bundled-npm(babel-core) = 6.11.4
-Provides: bundled-npm(to-fast-properties) = 1.0.2
-Provides: bundled-npm(source-map-support) = 0.2.10
-Provides: bundled-npm(ms) = 0.7.1
-Provides: bundled-npm(regenerator-runtime) = 0.9.5
-Provides: bundled-npm(brace-expansion) = 1.1.6
-Provides: bundled-npm(ansi-styles) = 2.2.1
+Provides: bundled-npm(core-js) = 2.5.1
+Provides: bundled-npm(source-map-support) = 0.4.18
+Provides: bundled-npm(invariant) = 2.2.2
+Provides: bundled-npm(babel-generator) = 6.26.0
+Provides: bundled-npm(globals) = 9.18.0
+Provides: bundled-npm(ms) = 2.0.0
+Provides: bundled-npm(regenerator-runtime) = 0.11.0
+Provides: bundled-npm(to-fast-properties) = 1.0.3
+Provides: bundled-npm(brace-expansion) = 1.1.8
 Provides: bundled-npm(has-ansi) = 2.0.0
+Provides: bundled-npm(escape-string-regexp) = 1.0.5
+Provides: bundled-npm(ansi-styles) = 2.2.1
 Provides: bundled-npm(strip-ansi) = 3.0.1
 Provides: bundled-npm(supports-color) = 2.0.0
-Provides: bundled-npm(user-home) = 1.1.1
-Provides: bundled-npm(os-tmpdir) = 1.0.1
-Provides: bundled-npm(get-stdin) = 4.0.1
+Provides: bundled-npm(os-tmpdir) = 1.0.2
+Provides: bundled-npm(os-homedir) = 1.0.2
+Provides: bundled-npm(loose-envify) = 1.3.1
 Provides: bundled-npm(minimist) = 0.0.8
-Provides: bundled-npm(loose-envify) = 1.2.0
-Provides: bundled-npm(repeating) = 1.1.3
-Provides: bundled-npm(minimist) = 1.2.0
-Provides: bundled-npm(escape-string-regexp) = 1.0.5
-Provides: bundled-npm(minimatch) = 3.0.2
+Provides: bundled-npm(detect-indent) = 4.0.0
+Provides: bundled-npm(jsesc) = 1.3.0
+Provides: bundled-npm(trim-right) = 1.0.1
 Provides: bundled-npm(concat-map) = 0.0.1
-Provides: bundled-npm(ansi-regex) = 2.0.0
-Provides: bundled-npm(source-map) = 0.1.32
-Provides: bundled-npm(js-tokens) = 1.0.3
-Provides: bundled-npm(is-finite) = 1.0.1
-Provides: bundled-npm(amdefine) = 1.0.0
-Provides: bundled-npm(balanced-match) = 0.4.2
-Provides: bundled-npm(number-is-nan) = 1.0.0
-AutoReq: no
-AutoProv: no
+Provides: bundled-npm(balanced-match) = 1.0.0
+Provides: bundled-npm(ansi-regex) = 2.1.1
+Provides: bundled-npm(repeating) = 2.0.1
+Provides: bundled-npm(is-finite) = 1.0.2
+Provides: bundled-npm(number-is-nan) = 1.0.1
+AutoReq: no 
+AutoProv: no 
 
 
 %description
-Babel compiler core.
-
-%package doc
```

Comment:
Is removal of the -doc package intentional? Should this spec also obsolete it then?